### PR TITLE
Adds Jest + React Testing Library 🧑‍🔬

### PR DESCRIPTION
**Pull request by rosschapman**
After a short pairing session today, @adriencyberspace and I got Jest installed and a simple unit test (`strings.test.ts`) running. I took it further and added RTL and migrated an existing test onto our new toolchain. Also updates docs.

### CODE REVIEW

File: app/components/listing/__tests__/RelativeOpeningTimes.spec.tsx
Comment (rosschapman) [inline app/components/listing/__tests__/RelativeOpeningTimes.spec.tsx:1]: Put on ice in Notion: https://www.notion.so/exygy/RelativeOpeningTime-tests-386441080de04860a2d581f6bf434e55?pvs=4

File: app/models/RecurringSchedule.ts [hunk 1]
```diff
@@ -2,9 +2,9 @@
  * Data structures used for consistently representing time throughout the app.
  */
 
-import { invert } from "utils/arrays";
 import sortBy from "lodash.sortby";
 import minBy from "lodash.minby";
+import { invert } from "../utils/arrays";
```
Comment (rosschapman) [inline app/models/RecurringSchedule.ts:7]: Jest complained about this import although it appeared to be working in source code.

File: README.md [hunk 2]
```diff
@@ -126,6 +126,31 @@ There are two protected branches - development and main. Main is the default bra
 
 There are two live instances - a [staging instance](https://our415-staging-a91cdc6d7b2b.herokuapp.com/) and a [production instance](https://our415-abb7eecb7449.herokuapp.com/). Merges onto the development branch deploys the development branch to the staging isntance. Merges onto the main branch deploys the main branch to the production instance. See the [github workflows](https://github.com/Exygy/askdarcel-web/tree/main/.github/workflows) for the details. Environment variables used in the deployments are set in github environments - 'prod' supplies the production instance and 'dev' supplies the staging instance..
 
+## Testing
```
Comment (adriencyberspace) [inline README.md:129]: Thanks for doing this!

**Review by adriencyberspace — APPROVED** [review]
🧪👍

**Pull request merged**